packages/client3/lib/graphql/rest-api-schema.graphql

```diff
@@ -0,0 +1,31 @@
+type Pool {
+  id: String!
+  poolName: String!
+  projectDetail: String!
+  goalAmount: String!
+  closingDate: String!
+  projectCoverImage: String
+  walletAddress: String!
+  terms: Terms
+  status: POOL_STATUS_TYPE!
+  transactionHash: String
```

Comment:
This is addressed in 138 but `transactionHash` to `poolAddress` 

Either we can leave it and change it later with 138 or do it now.

Reply:
Have ya changed it in 138?

Reply:
Yes

Reply:
Awesome thank you...Then we will add related changes there